galite-core/src/main/kotlin/org/kopi/galite/ui/vaadin/block/Block.kt

```diff
@@ -348,6 +348,7 @@ abstract class Block(private val droppable: Boolean) : Div(), HasEnabled {
         //dndWrapper = DragAndDropWrapper(layout) TODO
         //dndWrapper.setImmediate(true)
         //setContent(dndWrapper)
+        setContent(layout as Component)
       } else {
         setContent(layout as Component)
       }
```

Comment:
can we change code to :  because setContent(layout as Component) is in the if and the else
```
 if (layout == null) {
      layout = createLayout()
      if (droppable) {
        //dndWrapper = DragAndDropWrapper(layout) TODO
        //dndWrapper.setImmediate(true)
        //setContent(dndWrapper)
      } 
      setContent(layout as Component)
    }
```